db/data.py

```diff
@@ -460,3 +461,40 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def check_low_level_duplicates(data):
+    """
+        Check if data comes from an audio file already analysed
+        Uses the md5_encoded field
+    Args: data
+        A dictionary containing the low-level data extracted using essentia
```

Comment:
And then the name of the function can be more explicit like `find_md5_duplicate`. Perhaps even accept just that hash and return existing ID if it was found. It should not create a new one, just that one thing for finding a duplicate.